tcptracer-bpf.c

```diff
@@ -538,6 +538,9 @@ int kprobe__tcp_set_state(struct pt_regs *ctx)
 		if (!read_ipv4_tuple(&t, status, skp)) {
 			return 0;
 		}
+		if (state == TCP_CLOSE) {
+			bpf_map_delete_elem(&tuplepid_ipv4, &t);
```

Comment:
Could you add `return 0;` after deleting the entry from the map? It'll make it more clear that it will not emit a dummy "connect" event with `bpf_perf_event_output` line 570 in that case. The current patch works because it relies on the test line 547-549 to do the `return 0;`... but it's less readable IMO.

Ditto for the IPv6 branch.